ecosystem/typescript/sdk/src/aptos_types/type_tag.ts

```diff
@@ -386,6 +386,12 @@ export class TypeTagParser {
         bail("Invalid type tag.");
       }
 
+      // an Object `0x1::object::Object<T>` doesn't hold a real type, it points to an address
+      // therefore, we parse it as an address and dont need to care/parse the `T` type
+      if (module === "object" && name === "Object") {
+        return new TypeTagAddress();
+      }
+
```

Comment:
ah, this will break the TypeParser. Try parsing below type.
`0x1::some_module::SomeResource<0x4::object::Object<T>, 0x1::aptos_coin::AptosCoin>`